dts/bindings/modem/quectel,bg9x.yaml

```diff
@@ -20,3 +19,11 @@ properties:
 
   mdm-wdisable-gpios:
     type: phandle-array
+
+  autostarts:
```

Comment:
`always-on`? Just an idea

Reply:
You should probably keep this addition, it seems useful. Also if it matches names in source, that name is good, keep it